docs/nodes/mamaki.md

```diff
@@ -1,25 +1,25 @@
-# DevNet-2
+# Mamaki
 
-This guide contains the relevant sections for how to connect to Devnet,
-depending on the type of node you are running. Devnet-2 is a milestone
+This guide contains the relevant sections for how to connect to Mamaki,
+depending on the type of node you are running. Mamaki is a milestone
 in Celestia, allowing everyone to test out core functionalities on the
-network. You can read more on the announcement [here](https://blog.celestia.org/celestia-launches-devnet/)
+network.
 
 Your best approach to participating is to first determine which node
 you would like to run. Each node guides will link to the relevant network
 in order to show you how to connect to them.
 
 You have a list of options on the type of nodes you can run in order to
-participate in Devnet-2:
+participate in Mamaki:
```

Comment:
@YazzyYaz can you add a link to full node and make a separation between them like: 


```
Consensus Node: 
* [Validator Node](../nodes/validator-node.md)
DA Node:
* [Bridge Node](../nodes/bridge-node.md)
* [Full Node](blah blah)
* [Light Node](../nodes/light-node.md)
```

Reply:
I can make a separate tutorial for a full-node. How does one run a full-node?

Reply:
@renaynay I moved these suggestions and the new Full Node tutorial to a separate PR https://github.com/celestiaorg/docs/pull/87